src/gui/keymap.cpp

```diff
@@ -1273,6 +1322,9 @@ QString KeyMap::friendlyName(const QString& key, Layout layout){
     if(map.contains(key))
         return map[key].friendlyName();
     map = KeyMap(HARPOON, layout);
+    if(map.contains(key))
```

Comment:
These changes are not needed

Reply:
Are you sure? As I understand it this is the lookup for getting human-friendly name for mouse-specific keys?